sdk/lib/crt/misc/i10output.c

```diff
@@ -1,5 +1,140 @@
 #include <precomp.h>
 
+// From Wine msvcrt.h
+typedef struct {ULONG x80[3];} MSVCRT__LDOUBLE; /* Intel 80 bit FP format has sizeof() 12 */
```

Comment:
Would it make sense to add a `static_assert` to validate that the type has the expected size?

Reply:
Is it necessary to have an `MSVCRT__LDOUBLE` and a separate `_LDOUBLE` structure (stdlib.h)? Is the latter one a "standard" definition that doesn't actually match what MSVC does?

Reply:
It is 3 ULONGs, I think the size should be obvious. t's also only used in this file.
We need this, because the one from stdlib is different.